GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifyAccountDetail.m

```diff
@@ -27,6 +27,30 @@
 
 @implementation GIDVerifyAccountDetail
 
+- (nullable instancetype)initWithConfig:(GIDConfiguration *)configuration {
+  self = [super init];
+  if (self) {
+    _configuration = configuration;
+  }
+  return self;
+}
+
+- (nullable instancetype)init {
```

Comment:
Please add this to the `.h` and document what the initializer does, including how it can return nil.

Reply:
Done.